docs/toc.md

```diff
@@ -77,6 +77,8 @@
 ### [Analyzing third-party dependencies](core/porting/third-party-deps.md)
 ### [Porting libraries](core/porting/libraries.md)
 <!--### [🔧 NuGet packages](core/porting/nuget-packages.md)-->
+## [Building .NET Core](core/building/index.md)
```

Comment:
Building -> Build 

would it help to clarify what you're building? Build .NET Core from source, for example.